docs/examples.rst

```diff
@@ -0,0 +1,199 @@
+Examples
+--------
+
+Get Raw Data
+^^^^^^^^^^^^
+
+Initialize a ``DataMonster`` object:
+
+..  code::
+
+    dm = DataMonster(<key_id>, <secret_key>)
+
+Initialize a ``Datasource`` object (we will use a small data source from the provider Earnest for the purposes of this example):
+
+..  code::
+
+    ds = dm.get_datasource_by_name('Earnest Blended Index Panel Sales YoY')
```

Comment:
let's confirm with Kevin if we can have this in our official docs - perhaps instead it should be <data source name> or <data source id>. or maybe just change to `1010 Debit Sales Index` since that was already in the readme. the point is that some providers may not want it to be publicly known that they are on our platform